logging/structured/src/main/scala/tofu/logging/LoggableInstances.scala

```diff
@@ -0,0 +1,192 @@
+    override def putField[I, V, R, M](a: Short, name: String, input: I)(implicit receiver: LogRenderer[I, V, R, M]): R =
+      receiver.addInt(name, a.toLong, input)
+  }
+
+  final implicit val intLoggable: Loggable[Int] = new SingleValueLoggable[Int] {
+    def logValue(a: Int): LogParamValue                                                                              = IntValue(a.toLong)
+    override def putField[I, V, R, M](a: Int, name: String, input: I)(implicit receiver: LogRenderer[I, V, R, M]): R =
+      receiver.addInt(name, a.toLong, input)
+  }
+
+  final implicit val longLoggable: Loggable[Long] = new SingleValueLoggable[Long] {
+    def logValue(a: Long): LogParamValue                                                                              = IntValue(a)
+    override def putField[I, V, R, M](a: Long, name: String, input: I)(implicit receiver: LogRenderer[I, V, R, M]): R =
+      receiver.addInt(name, a, input)
+  }
+
+  final implicit val bigIngLoggable: Loggable[BigInt] = new SingleValueLoggable[BigInt] {
+    def logValue(a: BigInt): LogParamValue = BigIntValue(a)
+    override def putField[I, V, R, M](a: BigInt, name: String, input: I)(implicit
+        receiver: LogRenderer[I, V, R, M]
+    ): R                                   =
+      receiver.addBigInt(name, a, input)
+  }
+
+  final implicit val bigDecimalLoggable: Loggable[BigDecimal] = new SingleValueLoggable[BigDecimal] {
+    def logValue(a: BigDecimal): LogParamValue = DecimalValue(a)
+    override def putField[I, V, R, M](a: BigDecimal, name: String, input: I)(implicit
+        receiver: LogRenderer[I, V, R, M]
+    ): R                                       =
+      receiver.addDecimal(name, a, input)
+  }
+
+  final implicit val floatLoggable: Loggable[Float] = new SingleValueLoggable[Float] {
+    def logValue(a: Float): LogParamValue                                                                              = FloatValue(a.toDouble)
+    override def putField[I, V, R, M](a: Float, name: String, input: I)(implicit receiver: LogRenderer[I, V, R, M]): R =
+      receiver.addFloat(name, a.toDouble, input)
+  }
+
+  final implicit val doubleLoggable: Loggable[Double] = new SingleValueLoggable[Double] {
+    def logValue(a: Double): LogParamValue = FloatValue(a)
+    override def putField[I, V, R, M](a: Double, name: String, input: I)(implicit
+        receiver: LogRenderer[I, V, R, M]
+    ): R                                   =
+      receiver.addFloat(name, a, input)
+  }
+
+  final implicit val booleanLoggable: Loggable[Boolean] = new SingleValueLoggable[Boolean] {
+    def logValue(a: Boolean): LogParamValue = BoolValue(a)
+    override def putField[I, V, R, M](a: Boolean, name: String, input: I)(implicit
+        receiver: LogRenderer[I, V, R, M]
+    ): R                                    =
+      receiver.addBool(name, a, input)
```

Comment:
This formatting hurts my eyes. Let's do smt about it.

Reply:
there is issue about it https://github.com/tofu-tf/tofu/issues/292